packages/cli/src/ui/utils/clipboardUtils.ts

```diff
@@ -21,6 +22,9 @@ export const IMAGE_EXTENSIONS = [
   '.heif',
 ];
 
+/** Matches strings that start with a path prefix (/, ~, or .) */
+const PATH_PREFIX_PATTERN = /^[/~.]/;
```

Comment:
Does this need to also work on Windows?

Windows paths typically take the form `C:\`, but they can also take the form `\\servername\path` or even `\\\\?\\C:\\very\\long\\path\\name.txt` when working around max path limitations.